### PR TITLE
add images

### PR DESCRIPTION
**Pull request by shivamSspirit**

### CODE REVIEW

**Review by GeetaPogul — APPROVED** [review]
used good class names ! 
good

File: Components/images/image.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+<div class="res-square"><img class="round-2" src="../../images/jpeg/cash.jpg" alt="" /></div>
+</div>
+<div class="img-code">
+<pre class="prettyprint lang-html" id='img-0'>
+<code>
+<div class="round-container">
+<div class="res-square">
+<img class="round-1" src="../../images/jpeg/bird.jpg" alt="" />
+</div>
+<div class="res-square"><img class="round-2" src="../../images/jpeg/cash.jpg" alt="" /></div>
```
Comment (amirnathani9) [inline Components/images/image.html:31]: try to give clear and readable class name

File: Components/images/image.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+			// insert into PRE
+			document.getElementById("img-0").innerHTML = imgHtml;
+
+imgHtmlOne = imgHtmlOne.replace(
+				/&lt;script src[\s\S]*?&gt;&lt;\/script&gt;|&lt;!--\?[\s\S]*?--&gt;|&lt;pre\b[\s\S]*?&lt;\/pre&gt;/g,
+				'<span class="operative">$&<\/span>'
+			);
+			// insert into PRE
+			document.getElementById("img-1").innerHTML = imgHtmlOne;
+
```
Comment (amirnathani9) [inline Components/images/image.html:85]: make another file for javascript and link here

**Review by amirnathani9 — APPROVED** [review]
Looks good. Some suggestions are 
use the proper class name, 
do proper indentation,
 add a description with what changes you have done, screenshot and preview link

**Comment by sah-shah7** [comment]
Looks good, class naming system is good. Could add a description of img in alt attribute

**Pull request merged**